src/desktop/apps/auth2/components/ModalContainer.tsx

```diff
@@ -39,6 +42,30 @@ export const ModalContainer: React.SFC<any> = () => {
       }}
       csrf={sd.CSRF_TOKEN}
       handleSubmit={handleSubmit as any}
+      onSocialAuthEvent={data => {
+        const analyticsOptions = {
+          action:
+            data.mode === 'signup'
+              ? 'Created account'
+              : 'Successfully logged in',
+          type: data.mode,
+          context_module: data.contextModule,
+          modal_copy: data.copy,
+          trigger: data.trigger || 'click',
+          trigger_seconds: data.triggerSeconds,
+          intent: data.intent,
+          auth_redirect: data.redirectTo || data.destination,
+          service: data.service,
+        }
+
+        Cookies.set(
+          `analytics-${data.mode}`,
```

Comment:
The major change is here. I decided two use two distinct cookies for analytics tracking `analytics-signup` and `analytics-login`. I think this makes think a little more clear with the separation.

Reply:
Cool that's fine, but that also means we have to be more careful with clearing the right cookies after using them in the legacy code right?

Reply:
Yeah, that is in part why I separated them. I wanted to ensure we cleared out the correct one. When they weren't separated I had to use a bunch of conditional logic to ensure I was expiring the right one.

Reply:
I can walk you through it IRL.